### DISCUSSION TITLE
random_state should not be set in productive use

### DISCUSSION THREAD

**Discussion by sly8787** [pycaret/pycaret#2387] [Q&A]
Hi guys,

is there a way to prevent the random_state from being set? set_config('seed', None) unfortunately did not work!

```python
AdaBoostClassifier(algorithm='SAMME.R', base_estimator=None, learning_rate=1.0,
                   n_estimators=50, random_state=4053)
Transformation Pipeline and Model Successfully Saved
Transformation Pipeline and Model Successfully Loaded
Pipeline(memory=None,
         steps=[('dtypes',
                 DataTypes_Auto_infer(categorical_features=[],
                                      display_types=True, features_todrop=[],
                                      float_dtype='float32', id_columns=[],
                                      ml_usecase='classification',
                                      numerical_features=[],
                                      target='target', time_features=[])),
                ('imputer',
                 Simple_Imputer(categorical_strategy='not_available',
                                fill_value_categorical='not_available',
                                fi...
                 Advanced_Feature_Selection_Classic(ml_usecase='classification',
                                                    n_jobs=-1,
                                                    random_state=4053,
                                                    subclass='multi',
                                                    target='target',
                                                    top_features_to_pick=0.19999999999999996)),
                ('fix_multi', 'passthrough'), ('dfs', 'passthrough'),
                ('pca', 'passthrough'),
                ['trained_model',
                 AdaBoostClassifier(algorithm='SAMME.R', base_estimator=None,
                                    learning_rate=1.0, n_estimators=50,
                                    random_state=4053)]],
         verbose=False)
Transformation Pipeline and Model Successfully Loaded
```

Thanks for your help,

Best regards
